src/search/documents/paper.py

```diff
@@ -28,14 +28,28 @@ class PaperDocument(BaseDocument):
     paper_publish_date = es_fields.DateField()
     doi = es_fields.TextField(analyzer="keyword")
     openalex_id = es_fields.TextField()
-    # TODO: Deprecate this field once we move over to new app. It should not longer be necessary since authors property will replace it.
+    abstract = es_fields.TextField(analyzer=content_analyzer)
+    is_open_access = es_fields.BooleanField()
+    # TODO: Deprecate this field once we move over to new app.
+    # It should not longer be necessary since authors property will replace it.
     raw_authors = es_fields.ObjectField(
         properties={
             "first_name": es_fields.TextField(),
             "last_name": es_fields.TextField(),
             "full_name": es_fields.TextField(),
         },
     )
+    hubs = es_fields.ObjectField(
```

Comment:
can we add namespace to `hubs` while we are doing this?

Also - would we be able to query this easily? Can you provide a sample query of getting papers by hubs?